app/lib/katello/util/cveak_migrator.rb

```diff
@@ -0,0 +1,37 @@
+module Katello
+  module Util
+    class FakeActivationKey < ApplicationRecord
+      self.table_name = 'katello_activation_keys'
+    end
+
+    class CVEAKMigrator # used in db/migrate/20220929204746_add_content_view_environment_content_facet.rb
```

Comment:
As discussed please change the logic here so that

if the AK doesn't have CV or LCE, don't add it. (AKs aren't required to have cv/lce.)
if the AK has only CV, add it as Default LCE and that CV (and add a warning in the docs)
if the AK has only LCE, add it as Default LCE / Default Org View
if the AK has both, great, add that CVE

Reply:
Should be addressed in the latest commit

Reply:
if the AK has only CV, add it as Default LCE and that CV (and add a warning in the docs)
if the AK has only LCE, add it as Default LCE / Default Org View

^ These cases don't seem to be addressed yet? The code in CVEAKMigrator handles when a cv/lce ID combo doesn't have a corresponding ContentViewEnvironment object, but doesn't handle when one of the fields is omitted.

Reply:
Those CVEs should be part of the `aks_with_no_cve + aks_with_missing_cve` list. Seems to be ok to me? I am I missing something?